test/integrations/routes/users.test.js

```diff
@@ -21,27 +24,33 @@ describe('API endpoint: /api/users', () => {
   beforeEach(() => {
```

Comment:
I really need to understand why you have to call `beforeEach` method to drop table and not put that logic in `after`

Reply:
like this  after(() => models.Profile.destroy({ truncate: true }));

Reply:
Please remove the sync block too, we have a migration script already

Reply:
Thanks for the feedback @aanchirinah

It doesn't actually drop the table, what it simply does is that it clears the data on the database table `models.User.destroy()`. One of the benefits of doing is that, it prepares the database table regardless of whatever test cases may be introduced later in the future.

And every test cases is not dependant on another, which ensures that all test will always run on a clean, empty table.

Reply:
In regards to that, the previous comment prior to this PR states that `// drop tables`, that was a typo. However, it has been adjusted to pass the right message.

Reply:
I hope I have been able to tailor this in the right fashion (^_^). Looking forward to your feedback response, thanks.

Reply:
Oh yes @fejiroofficial, you are very correct, thanks for pointing that, running the migration script has covered that. I will make a quick adjust to that, thanks.

Reply:
Okay good one

Reply:
Hi @aanchirinah I have attended to your changes.

Reply:
@devdbrandy Why have you removed both hooks?